sidebars.js

```diff
@@ -141,17 +141,36 @@ const sidebars = {
     },
     {
       type: "category",
-      label: "Customize",
+      label: "Customize your Subnet",
       collapsed: true,
       items: [
         "subnets/customize-a-subnet",
+        "subnets/cross-subnet-communication"
+      ],
+    },
+    {
+      type: "category",
+      label: "Subnet-EVM Chain",
+      collapsed: true,
+      items: [
+        "subnets/customize-a-subnet",
+        "subnets/subnet-evm-genesis",
+        "subnets/subnet-evm-precompiles",
+        "subnets/subnet-evm-avalanchego-config",
+        "subnets/subnet-evm-upgrades",
         "subnets/hello-world-precompile-tutorial",
+      ],
+    },
+    {
+      type: "category",
+      label: "Customize a VM Chain",
```

Comment:
As my previous comment, this should be changed to "Build a VM Chain", as most of the docs are "How to build ...."